components/expression_language/syntax.rst

```diff
@@ -127,18 +127,55 @@ Working with Functions
 ----------------------
 
 You can also use registered functions in the expression by using the same
-syntax as PHP and JavaScript. The ExpressionLanguage component comes with one
-function by default: ``constant()``, which will return the value of the PHP
-constant::
+syntax as PHP and JavaScript. The ExpressionLanguage component comes with the
+following functions by default:
+
+* ``constant()``
+* ``enum()``
+
+Constant function
```

Comment:
```suggestion
`constant()` function
```
?

Reply:
I agree this is better! I'm doing the change, and I'll revert it if Wouter and/or Javier don't agree! 🙂